ppa/archive/solr.py

```diff
@@ -128,11 +128,26 @@ class ArchiveSearchQuerySet(SolrQuerySet):
         "image_id_s",
         "first_page_i",
         "source_url",
+        "work_type_s",
     ]
+    # aliases for any fields we want to rename for search and display
+    # (must also be included in return_fields list)
+    aliases = {
+        "source_t": "source",
+        "image_id_s": "image_id",
+        "first_page_i": "first_page",
+        "work_type_s": "work_type",
+    }
```

Comment:
i think some of these get referenced in other open PRs so the merge order might matter (unless you can still access a field via its original un-aliased name)

Reply:
thanks, I'll keep an eye on that when I merge — I think this one should go first but will be good to confirm!